src/libraries/System.Runtime.Intrinsics.Experimental/System.Runtime.Intrinsics.Experimental.sln

```diff
@@ -0,0 +1,85 @@
+﻿Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Based on the conversation with @jkotas and @terrajobst, we shouldn't add `System.Runtime.Intrinsics.Experimental` back.

Instead, we should expose this from the normal `System.Runtime.Intrinsics` reference assembly and ensure it has the `[RequiresPreviewFeatures]` attribute I mentioned above.

Reply:
Ok, I will move this back to the normal System.Runtime.Intrinsics.